l10n_it_fatturapa_pec/readme/CONFIGURE.rst

```diff
@@ -4,8 +4,20 @@ Consultare il modulo `l10n_it_sdi_channel`.
 
 Lo stato dell'esportazione XML può essere forzato impostando 'Permettere di forzare lo stato dell'esportazione e-fattura' nelle impostazioni tecniche dell'utente.
 
+Per ogni azienda, in
+
+Contabilità → Configurazione → Impostazioni → Fattura elettronica
+
+specificare l'utente che sarà utilizzato come creatore delle e-fatture fornitore create dalla PEC.
+
 **English**
 
 See `l10n_it_sdi_channel` module.
 
-XML export state can ba forced setting 'Allow to force e-invoice export state' in user's technical settings.
+XML export state can ba forced setting 'Allow to force the supplier e-bill export state' in user's technical settings.
+
+For every company, in
+
+Accounting → Configuration → Settings → Fattura elettronica
```

Comment:
E qui va in inglese, la voce di menù è "Electronic Invoices"

Reply:
Ok!